doc/conf.py

```diff
@@ -53,7 +53,7 @@
 
 # Extlinks extension
 extlinks = {
-    "repo":  ("https://github.com/zeek/spicy/blob/main/%s", "#%s"),
+    "repo":  ("https://github.com/zeek/spicy/blob/v%s/%%s" % release, "#%s"),
```

Comment:
@rsmmr, this is really the most interesting bit in this PR. This always links to artifacts in the latest release.